### DISCUSSION TITLE
Rebroadcast ASTERIX format

### DISCUSSION THREAD

**Discussion by larswilliam** [vradarserver/vrs#35] [Ideas]
Hi,
Thanks for awesome functionality.
Would it be to much effort to rebroadcast the ASTERIX format as well?

Best regards
Lars William

**Comment by vradarserver** [comment]
Sorry for the delay in replying. I don't think the format itself would be too problematic, I remember seeing the spec for it before and while it's a bit involved it's not too bad. It would mean adding support for UDP connections to VRS, but that wouldn't be a bad thing either.

The issue is more that I don't have a connection to test it with. If you have an ASTERISK receiver that I can connect to over the Internet without having to VPN onto third party networks then that would be cool, otherwise it'll need to wait until such a thing becomes available.
